### DISCUSSION TITLE
I'm a newbie, how to run it?

### DISCUSSION THREAD

**Discussion by OwenCleon** [vfsfitvnm/frida-il2cpp-bridge#304] [Q&A]
Hey, sorry if this is a dumb question, but how to run it?

**Comment by commonuserlol** [comment]
install nodejs, create your project dir, install `@types/frida-gum` via npm, install `frida-il2cpp-bridge` also via npm. if you have frida python just exec `frida-compile index.ts -o index.js`, if no - install `frida-compile` via npm and run `npx frida-compile index.ts -o index.js`. and finally run frida (i hope you know how to do this)

**Reply by QPLAYid** [reply]
And how trace it 
// Assembly-CSharp
class PlayerController : UnityEngine.MonoBehaviour
{

**Reply by QPLAYid** [reply]
Trace class name  Assembly-CSharp

**Comment by bluewave41** [comment]
The wiki page documents a lot of it https://github.com/vfsfitvnm/frida-il2cpp-bridge/wiki/Installation

You'll reference it a lot.
